biojava-aa-prop/pom.xml

```diff
@@ -105,12 +105,15 @@
 		</dependency>
 
 		<dependency>
-			<groupId>javax.xml.bind</groupId>
-			<artifactId>jaxb-api</artifactId>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>4.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.glassfish.jaxb</groupId>
 			<artifactId>jaxb-runtime</artifactId>
+			<version>4.0.3</version>
+			<scope>runtime</scope>
```

Comment:
Please remove version and scope here. It should be only in `dependencyManagement` in top pom.xml

Reply:
Removed explicit versions and scope from POM as required.